packages/super-components/components/msc-tail-logs/src/msc-tail-logs/translations.ts

```diff
@@ -0,0 +1,41 @@
+import { Language } from '@ovhcloud/msc-utils';
+
+export type Translations = {
+  empty_logs: string;
+  refresh_button_label: string;
+  service_error_message: string;
+};
+
+/**
+ * Import dynamically the right translation file
+ */
+export async function getTranslations(
+  language?: Language,
+): Promise<Translations> {
+  try {
+    switch (language) {
+      case 'de-DE':
+        return await import('../translations/Messages_de_DE.json');
+      case 'en-GB':
+        return await import('../translations/Messages_en_GB.json');
+      case 'es-ES':
+        return await import('../translations/Messages_es_ES.json');
+      case 'fr-CA':
+        return await import('../translations/Messages_fr_CA.json');
+      case 'fr-FR':
+        return await import('../translations/Messages_fr_FR.json');
+      case 'it-IT':
+        return await import('../translations/Messages_it_IT.json');
+      case 'pl-PL':
+        return await import('../translations/Messages_pl_PL.json');
+      case 'pt-PT':
+        return await import('../translations/Messages_pt_PT.json');
+      default:
+        return await import('../translations/Messages_fr_FR.json');
+    }
+  } catch {
+    throw new Error(`No translations found for language ${language}`);
+  }
```

Comment:
**nitpick:** This logic can be utilised by lot of components. So move it to a place from where it can be injected in all components and reused.

Reply:
It will be used by all components that need translations

Except that we can't centralize this logic because it uses dynamic import and we can't use variable inside the import string. So this is not possible to centralize this function because the import path depends on the component.